study.md

```diff
@@ -246,5 +256,12 @@ Pick a few design heuristics. Describe some common, named refactorings you can
 use approach a good design.
 
 ```md
-<!-- your answer here -->
+Making sure your code is DRY - if code is repeated, you may lose track of where
+it occurs, which opens up the possibility of not changing it everywhere you need
+to. Additionally, it lengthens your code without adding any additional benefits,
+which  makes it harder to follow and maintain. Making sure code only has one
+responsibility - if each thing is reponsible for one task, you can know in a
+glance whether it's working or not... there's no logic tree to go down. This
+allows for easier-to-follow code that lends itself to readability and
+maintenance.
```

Comment:
Great start. What would you use to DRY up duplicated logic? Maybe "Extract Method". Have a look at some named refactorings, like "Extract Method", to level up.